### DISCUSSION TITLE
Question about sorry txt and other ways to apologize

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#8276] [QA - tech support + general] [answered]
I know that the game has sorry text, if I'm not mistaken it is given in the case when Monika wants to leave. What happens if you give it when everything is fine? Will she react to it or not? And if so, tell me how to make such a letter. I also have a question about other ways of apologizing. Is it possible to apologize for something in any other way?

**Comment by dreamscached** [comment, marked as answer]
There is an in-game menu *You > I want to apologize*
